packages/dapp/src/types/internal.types.ts

```diff
@@ -64,7 +55,7 @@ export interface ChainsMenuListItem {
   showMoreIcon?: boolean;
   checkIcon?: boolean;
   onClick?: any;
-  chainId: ChainId;
+  chainId: number;
```

Comment:
Is this legit? Still not sure about it

Reply:
Okay, you said this multiple times, thanks for you patience, will bring this back!